text/0000-fabric-private-chaincode-lite.md

```diff
@@ -41,17 +42,37 @@ Such data includes secret cryptographic keys, which the chaincode uses to secure
     and also still mention all possible use-cases. 
     The distinction is done later in the architecture/design section
 -->
+
+***TODO*** document is missing a bit in the intro section about:
+- clear statement what is the purpose of the document
+- link to full FPC
+- explain that we begin with FPC lite on our journey to FULL FPC security features
+- differentiate between FPC Lite/Full Security properties and the feature to support vanilla Fabric- clarify
```

Comment:
The first paragraph of the fpc lite architecture actually does do that. But i guess not clear enough?  Maybe pointing there also here to the TL-less google doc (which we only reference at end) as a source of more in-depth discussion on it?